docs/src/manual/manual.md

```diff
@@ -24,10 +24,7 @@ Description of the core of the Dionysos.jl package, the [`src`](https://github.c
 
 The system types supported in Dionysos.jl are:
 * [`MathematicalSystems`](https://juliareach.github.io/MathematicalSystems.jl/latest/lib/types/#MathematicalSystems.AbstractSystem), which proposes generic and flexible system definitions (e.g.     discrete-time/continuous-time, constrained, noisy systems), such that, for example, the system [`MathematicalSystems.NoisyConstrainedAffineControlDiscreteSystem`](https://juliareach.github.io/MathematicalSystems.jl/latest/lib/types/#MathematicalSystems.NoisyConstrainedAffineControlDiscreteSystem) of the form $$x(k+1) = A x(k) + B u(k) + c + D w(k), \ x(k)\in\mathcal{X}, \ u(k)\in\mathcal{U},\ w(k)\in\mathcal{W}\ \forall k$$
-where 
-    * $\mathcal{X}$ is the state constraints;
-    * $\mathcal{U}$ is the input constraints;
-    * $\mathcal{W}$ is the noise constraints.
+where $\mathcal{X}$ is the state constraint, $\mathcal{U}$ is the input constraint and $\mathcal{W}$ is the noise constraint.
```

Comment:
👍

Reply:
To format I used julia-1.8, seems that the problem is julia-1.6

Reply:
Which is weird because Actions is set up with Julia 1.3.0 😆